site/cypress/e2e/profile.spec.js

```diff
@@ -71,6 +71,59 @@ describe('Test cases revolving around user profile page', () => {
         cy.get('#secondary-email-notify-row').should('be.visible');
     });
 
+    // Selenium test_basic_info
+    it('Should start with accurate values', () => {
+        cy.get('[data-testid="username-row"]').should('contain.text', 'instructor');
+        cy.get('[data-testid="givenname-row"]').should('contain.text', 'Quinn');
+        cy.get('[data-testid="familyname-row"]').should('contain.text', 'Instructor');
+        cy.get('[data-testid="pronouns-row"]').should('contain.text', 'She/Her');
+        cy.get('[data-testid="email-row"]').should('contain.text', 'instructor@example.com');
+    });
+
+    // Selenium test_time_zone_selection
+    it('Time zone selector should work', () => {
+        cy.get('[data-testid="time-zone-dropdown"]').should('contain.text', 'NOT_SET/NOT_SET');
+        cy.get('[data-testid="time-zone-dropdown"]').select('(UTC-04:00) America/New_York');
+        // Since the login success message is still up, we get the next message.
+        cy.get('[data-testid="popup-message"]').next().should('contain.text', 'Time-zone updated successfully');
+        cy.get('[data-testid="time-zone-dropdown"]').select('NOT_SET/NOT_SET');
+    });
+
```

Comment:
So perhaps I a misunderstanding something, but in the Selenium tests I saw a test called "test_edit_preferred_names" that I expected to be in this cypress test at around this part of the code.

Reply:
This functionality is already tested with the `it('Should test the modifying of the values')` test, I believe